curp/src/server/raw_curp/mod.rs

```diff
@@ -328,6 +342,68 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         (info, Ok(()))
     }
 
+    /// Handle `propose_conf_change` request
+    pub(super) fn handle_propose_conf_change(
```

Comment:
This method looks similar to 'handle_propose`. Can we extract the common part of these two methods as a new method to avoid duplication?

Reply:
Fixed